lms/djangoapps/branding/__init__.py

```diff
@@ -57,11 +58,11 @@ def get_visible_courses(org=None, filter_=None):
         )
 
     if filtered_visible_ids:
-        return [course for course in courses if course.id in filtered_visible_ids]
+        return courses.filter(id__in = filtered_visible_ids)
```

Comment:
I think all the examples in this PR of removing the conversion of a queryset to a list to prevent the premature evaluation of the query could be a great short presentation at eng all hands like devops sometimes does with the devops minute.  
many of us myself included often forget to think about whether we're making needless queries when dealing with querysets since the abstraction layer means we don't have to think about it as much

Reply:
Yeah, will do.